Champions/Karthus/Q.cs

```diff
@@ -0,0 +1,72 @@
+using GameServerCore.Enums;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+
+namespace Spells
+{
+    public class KarthusLayWasteA1 : IGameScript
+    {
+        public void OnActivate(IChampion owner)
+        {
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            AddParticleTarget(owner, "Karthus_Base_Q_Hand_Glow.troy", owner, 1, "R_Hand");
+            AddParticle(owner, "Karthus_Base_Q_Point.troy", spell.X, spell.Y);
+            AddParticle(owner, "Karthus_Base_Q_Ring.troy", spell.X, spell.Y);
+            AddParticle(owner, "Karthus_Base_Q_Skull_Child.troy", spell.X, spell.Y);
+            CreateTimer(0.5f, () =>
+            {
+                IGameObject m = AddMinion(owner, "TestCube", "TestCube", spell.X, spell.Y);
+                var range = GetUnitsInRange(m, 100, true);
+                var ap = owner.Stats.AbilityPower.Total;
+                var damage = 20f + spell.Level * 20f + ap * 0.3f;
+                
+                foreach (var units in range)
+                {
+                    if (units.Team != owner.Team && units is IChampion || units is IMinion)
+                    {

```

Comment:
This block seems to have a lot of wrong things.
1. Double damage is applied if it hits *one* unit, not *two*.
2. `range` variable includes things which may be allied and/or things you cannot hit (e.g. wards, turrets or untargetable things). Using its count is problematic.

Reply:
about that part, I'll take a look,because current code I used "AddMinion", so it will count to 2.
I will use Particle instead, so It will be 1.
count part I will figure out how to do that.